yarn.lock

```diff
@@ -1,4 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-# yarn lockfile v1
```

Comment:
hrrm wait why is this being generated?

Reply:
@ebb-tide i removed since I had apparently introduced it by running yarn install at the root directory 7 months ago. Just came across it when I was working in fixing the issue that stemmed from the lock file and figured it should be removed to reduce confusion